src/Guide/Api/Methods.hs

```diff
@@ -204,4 +224,4 @@ dbUpdate db x = liftIO $ do
 dbQuery :: (MonadIO m, EventState event ~ GlobalState, QueryEvent event)
         => DB -> event -> m (EventResult event)
 dbQuery db x = liftIO $
-  Acid.query db x
+  Acid.query db x
```

Comment:
By convention, text files [should have trailing newlines](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file). In VSCode this is configured with `"files.insertFinalNewline": true`.